_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Two Months Of Tech Interviews 💼"
+comments: true
+categories: tech recurse job
+---
+
+The time finally came. 
+
+Full. Time. Job. Search.
+
+Over the last 5 years, I've accumulated over 2.5 years of paid software engineering experience. I've also accumulated a fair amount of self-directed experience.
+
+But, I've never accepted a full-time job...
+
+> Sorry, I have more school to finish.
+
+> Sorry, I want to work in a different city.
+
+> Sorry, I want to travel and focus on my own projects.
+
+> Sorry, I want to attend the Recurse Center.
+
+...until now.
+
+On Monday, I accepted a full-time offer with [Squarespace](https://www.squarespace.com/) 🎉.
+
+Job hunting was far from the seamless process I imagined it would be. Here's a review of my 2 months interviewing in tech.
+
+### Interview prep
+
+The first step of my journey was interview was prep.
```

Comment:
Extra `was` after interview. 

The first step of my journey was interview prep.